src/uvw/process.hpp

```diff
@@ -28,7 +28,7 @@ enum class UVProcessFlags: std::underlying_type_t<uv_process_flags> {
 
 
 enum class UVStdIOFlags: std::underlying_type_t<uv_stdio_flags> {
-    IGNORE = UV_IGNORE,
+    IO_IGNORE = UV_IGNORE,
```

Comment:
Documentation must be updated as well (see the `stdio` member methods of the class `ProcessHandle` in the same file).